vector/src/main/java/im/vector/app/features/spaces/SpaceListViewModel.kt

```diff
@@ -296,7 +296,7 @@ class SpaceListViewModel @AssistedInject constructor(@Assisted initialState: Spa
             communityGroups
         }
                 .execute { async ->
-                    val rootSpaces = session.spaceService().getRootSpaceSummaries()
+                    val rootSpaces = async.invoke().orEmpty().filter { it.flattenParentIds.isEmpty() }
```

Comment:
On my side this is equivalent, but I do not have any spaces with child space. I will create some

Reply:
Ok, it works as expected